.gitignore

```diff
@@ -9,3 +9,6 @@ node_modules
 *.pyc
 dist/
 .vscode/
+variable-type-experiments/.git/
```

Comment:
I don't think these git rules should be needed unless the `variable-type-experiments` folder is a git folder (which it shouldn't ;) )

Reply:
I looked at my local copy repo but couldnt find those files. I think I may have removed them.